tests_pycsob/test_api.py

```diff
@@ -1,54 +1,56 @@
 # coding: utf-8
-import os
 import json
+import os
 import pytest
+import responses
 from collections import OrderedDict
-from requests.exceptions import HTTPError
+from freezegun import freeze_time
 from unittest import TestCase
-from urllib3_mock import Responses
+from requests.exceptions import HTTPError
 
 from pycsob import conf, utils
 from pycsob.client import CsobClient
 
+
+BASE_URL = 'https://gw.cz'
 KEY_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), 'fixtures', 'test.key'))
 PAY_ID = '34ae55eb69e2cBF'
 
-responses = Responses(package='requests.packages.urllib3')
-
 
 class CsobClientTests(TestCase):
 
     def setUp(self):
         self.key = open(KEY_PATH).read()
         self.c = CsobClient(merchant_id='MERCHANT',
-                            base_url='https://gw.cz',
+                            base_url=BASE_URL,
                             private_key=KEY_PATH,
                             csob_pub_key=KEY_PATH)
 
     def test_client_init_can_take_key_string(self):
         client = CsobClient(merchant_id='MERCHANT',
-                            base_url='https://gw.cz',
+                            base_url=BASE_URL,
                             private_key=self.key,
                             csob_pub_key=self.key)
         assert client.key == self.key
         assert client.pubkey == self.key
 
     def test_client_init_can_take_key_path(self):
         client = CsobClient(merchant_id='MERCHANT',
-                            base_url='https://gw.cz',
+                            base_url=BASE_URL,
                             private_key=KEY_PATH,
                             csob_pub_key=KEY_PATH)
         assert client.key == self.key
         assert client.pubkey == self.key
 
+    @freeze_time()
     @responses.activate
     def test_echo_post(self):
         resp_payload = utils.mk_payload(self.key, pairs=(
             ('dttm', utils.dttm()),
             ('resultCode', conf.RETURN_CODE_OK),
             ('resultMessage', 'OK'),
         ))
-        responses.add(responses.POST, '/echo/', body=json.dumps(resp_payload),
+        responses.add(responses.POST, utils.mk_url(BASE_URL, '/echo/'), body=json.dumps(resp_payload),
```

Comment:
`responses` need full URL, not just a part of it - `urllib3-mock` used to take care of that, so it has to be changed.